kafl_fuzzer/common/config.py

```diff
@@ -247,22 +248,31 @@ def _parse_with_config(self, parser):
 
         config = confuse.Configuration('kafl', modname='kafl_fuzzer')
 
-        # check default config search paths
+        # check default config locations
         config.read(defaults=True, user=True)
 
+        # ENV based config
```

Comment:
What removing the `if` statement:
~~~Python3
            try:
                debug_print(f"Checking configuration at KAFL_CONFIG_FILE: {os.environ['KAFL_CONFIG_FILE']}")
                config.set_file(os.environ['KAFL_CONFIG_FILE'])
            except Exception as e:
                logger.error("Failed to read configuration at $KAFL_CONFIG_FILE:\n" + str(e))
                sys.exit(1)
~~~

Reply:
We want to check only if the env was set - but if set, I think we want to complain about parse error but also FileNotFound or similar.